Makefile

```
@@ -595,6 +595,11 @@ include scripts/get-flags.mk
 CUDA_CXXFLAGS := $(BASE_CXXFLAGS) $(GF_CXXFLAGS) -Wno-pedantic
 endif
 
+ifdef LLAMA_USE_CURL
```

The pattern is: `LLAMA_XXX=1 make` -> `-DLLAMA_USE_XXX`

```suggestion
ifdef LLAMA_CURL
```